database/factories/PlaceCategoryFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Faker\Generator as Faker;
+use Hedonist\Entities\Place\PlaceCategory;
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+$factory->define(PlaceCategory::class, function (Faker $faker) {
```

Comment:
i think we can abstract our test data for place category in PlaceCategoryTableSeeder.php. For example
```
public function run()
{
        PlaceCategory::query()->insert([
            'cafe',
            'restaurant',
            'lunch',
            'bar',
            'coffee',
            'pizzeria'
        ]);
}
```
it is enough for now.